src/Model/Post.php

```diff
@@ -256,6 +272,7 @@ public function init() {
 					'capability' => $this->post_type_object->cap->edit_posts,
 				],
 				'excerptRendered' => function() {
+					setup_postdata( $this->post );
```

Comment:
Why are you setting up the post data in the individual fields when it happens in the __construct already? Seems like this would hurt performance if querying multiple fields that set up the post data.

Reply:
See the previous comment. It's call cached now, we just have to reset the globals to play nice with deferred resolution